lib/ansiblelint/formatters/__init__.py

```diff
@@ -143,25 +116,13 @@ def _severity_to_level(severity: str) -> str:
 class ParseableSeverityFormatter(BaseFormatter):
 
     def format(self, match: "MatchError", colored: bool = False) -> str:
-        formatstr = u"{0}:{1}: [{2}] [{3}] {4}"
 
         filename = self._format_path(match.filename or "")
         linenumber = str(match.linenumber)
         rule_id = u"E{0}".format(match.rule.id)
         severity = match.rule.severity
         message = str(match.message)
 
-        if colored:
-            filename = colorize(filename, Color.filename)
-            linenumber = colorize(linenumber, Color.linenumber)
-            rule_id = colorize(rule_id, Color.error_code)
-            severity = colorize(severity, Color.error_code)
-            message = colorize(message, Color.error_title)
-
-        return formatstr.format(
-            filename,
-            linenumber,
-            rule_id,
-            severity,
-            message,
-        )
+        return (
+            f"[filename]{filename}[/]:{linenumber}: [[error_code]{rule_id}[/]] "
+            f"[[error_code]{severity}[/]] [dim]{message}[/]")
```

Comment:
Does this mean the output is always colorized now?

Reply:
Nope. Rich does auto fallback to plain when terminal is not available. This means that we are not supposed to have conditionals inside the code regarding if ansi foo else bar.